modules/mapping-api/src/main/java/cn/maxpixel/mcdecompiler/mapping/format/MappingFormats.java

```diff
@@ -178,4 +202,4 @@ public static boolean registerMappingFormat(MappingFormat<?, ?> format) {
     public static Set<String> getFormatNames() {
         return ObjectSets.unmodifiable(MAPPING_FORMATS.keySet());
     }
-}
\ No newline at end of file
+}
```

Comment:
what happened here?

Reply:
I think that is just from the added lines for the new format